### PR TITLE
Propose to clarify

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2605

@dariomws please review

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/disable-basic-authentication-in-exchange-online.md [hunk 1]
```diff
@@ -333,7 +339,10 @@ For detailed syntax and parameter information, see [Remove-AuthenticationPolicy]
 
 ### How do you know that you've successfully disabled Basic authentication in Exchange Online?
 
-To confirm that the authentication policy was applied to users:
+To confirm that an authentication policy was directly applied to users:
+
+> [!NOTE]
+> Take into account that a default authentication policy could be already configured, see [Configure the default authentication policy](#configure-the-default-authentication-policy) for details.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/disable-basic-authentication-in-exchange-online.md:345]:
```suggestion
> Take into account that a default authentication policy could be already configured. See [Configure the default authentication policy](#configure-the-default-authentication-policy) for details.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**